frontend/src/employee-frontend/components/vasu/sections/VasuEvents.tsx

```diff
@@ -66,16 +74,28 @@ export function VasuEvents({ document: { documentState, events } }: Props) {
         <ChipContainer>
           <VasuStateChip state={documentState} labels={i18n.vasu.states} />
         </ChipContainer>
-        {topSectionEvents.map((key) => (
-          <EventRow key={key} label={i18n.vasu[key]} date={events[key]} />
-        ))}
-      </ListGrid>
-      <HorizontalLine slim />
-      <ListGrid labelWidth={labelWidth}>
-        {bottomSectionEvents.map((key) => (
-          <EventRow key={key} label={i18n.vasu[key]} date={events[key]} />
-        ))}
+        <EventRow label={i18n.vasu.modifiedAt} date={modifiedAt} />
+        <EventRow label={i18n.vasu.eventTypes.PUBLISHED} date={lastPublished} />
```

Comment:
this should have a different label, "Viimeksi julkaistu huoltajalle", so that the difference with other publish times below is clear